src/Nethermind/Nethermind.Network/ProtocolsManager.cs

```diff
@@ -405,12 +405,15 @@ private void AddNodeToDiscovery(ISession session, P2PProtocolInitializedEventArg
 
         public void AddSupportedCapability(Capability capability)
         {
-            if (_capabilities.Contains(capability))
+            _capabilities.Add(capability);
```

Comment:
Why we don't check 
`if (_capabilities.Contains(capability))`
anymore ?

Reply:
No need after changing from `List<Capability>` to `HashSet<Capability>`. `Add(item)` for `HashSet` is `AddIfNotPresent(item)` actually. https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1.add?view=net-6.0

Reply:
Cool. Missed the change.